sys-block/parted/parted-3.2.recipe

```diff
@@ -77,36 +80,42 @@ BUILD_PREREQUIRES="
 	cmd:autoreconf
 	cmd:awk
 	cmd:gcc$secondaryArchSuffix
-	cmd:gettext
+	cmd:gettext$secondaryArchSuffix
 	cmd:gperf
 	cmd:grep
 	cmd:iconv
 	cmd:ld$secondaryArchSuffix
 	cmd:libtoolize$secondaryArchSuffix
 	cmd:make
-	cmd:makeinfo
 	cmd:pkg_config$secondaryArchSuffix
 	cmd:sed
 	"
 
+TEST_REQUIRES="
+	cmd:dd
+	"
+
 defineDebugInfoPackage parted$secondaryArchSuffix \
 	"$libDir"/libparted.so.$libVersion \
 	"$libDir"/libparted-fs-resize.so.$libFsResizeVersion
 
 BUILD()
 {
 	autoreconf
+	MAKEINFO=true \
 	runConfigure --omit-dirs sbinDir ./configure --sbindir=$commandBinDir \
```

Comment:
A tab is missing before runConfigure.

Reply:
I would agree if the command was **_before_** `runConfigure`. But here, the command **_is_** `runConfigure`. `MAKEINFO=true` is  just modifying that command's environmental variables.
So imo the `runConfigure` and the `MAKEINFO=true` should be aligned.
(We have several other recipes where we have this kind of style, and I'm not the only one to have this opinion ;)

BTW, the style I am using here is more `git log -p`_-friendly_ than if I added a tab in front of `runConfigure`.
So I'd like to keep this unchanged :) Thanks!

Reply:
Well, it looks like I'm not the only one to think it _does_ make sense :smile:
<Details>

https://github.com/haikuports/haikuports/blob/6b25575af0ee31ade6acc9278457d0f738ee88d2/dev-db/sqlcipher/sqlcipher-3.4.2.recipe#L67
https://github.com/haikuports/haikuports/blob/309ccdded07928a7a8e084fb09e5a5d4da825e61/dev-embedded/gcc6809/gcc6809-4.6.4.recipe#L75
https://github.com/haikuports/haikuports/blob/75a3cf5384f650242f4b8a1094ab305d33ed99e7/dev-lang/v8/v8-3.16.1.recipe#L70
https://github.com/haikuports/haikuports/blob/307a0073e11e17161ff2087dc3b48a37168c81bd/dev-libs/atf/atf-0.21.recipe#L80
https://github.com/haikuports/haikuports/blob/3ed845aa74d8dd08252999fed518e4bed67c91f0/dev-libs/libpurple/libpurple-2.12.0.recipe#L122
https://github.com/haikuports/haikuports/blob/0ac7ea21bb1daf5c72140525170809282b37a5d2/dev-scheme/guile/guile-2.2.3.recipe#L88
https://github.com/haikuports/haikuports/blob/ef1d6f58a1302168a71899b96d61b318d8211393/dev-util/kyua/kyua-0.13.recipe#L80
https://github.com/haikuports/haikuports/blob/fc0936b04009e713f9bf551851781c83f9f23871/dev-vcs/cvs/cvs-1.12.13.1.recipe#L74
https://github.com/haikuports/haikuports/blob/69769a3ac56e8829e3445cca7edb8b27a66cdda7/media-libs/libvpx/libvpx-1.7.0.recipe#L104
https://github.com/haikuports/haikuports/blob/598decaca607909fa18e95e718d0eed8df242cd0/net-dns/yaddns/yaddns-1.1.recipe#L65
https://github.com/haikuports/haikuports/blob/cf123ac56c6a47d7eed864421283c26eaeddf98e/net-ftp/lftp/lftp-4.8.3.recipe#L81
https://github.com/haikuports/haikuports/blob/25af4c48a6840489fe4e717afb3465a4b4c39207/net-misc/openssh/openssh-7.0p1.recipe#L109
https://github.com/haikuports/haikuports/blob/8a585d344a96375a8aa2659dbf8b55be2b4d592b/net-misc/openssh/openssh-7.1p1.recipe#L119
https://github.com/haikuports/haikuports/blob/8a585d344a96375a8aa2659dbf8b55be2b4d592b/net-misc/openssh/openssh-7.1p2.recipe#L119
https://github.com/haikuports/haikuports/blob/8a585d344a96375a8aa2659dbf8b55be2b4d592b/net-misc/openssh/openssh-7.2p2.recipe#L118
https://github.com/haikuports/haikuports/blob/9b6f809e57c0a5c5697671a53589979bb8765f20/net-misc/openssh/openssh-7.4p1.recipe#L117
https://github.com/haikuports/haikuports/blob/fc0936b04009e713f9bf551851781c83f9f23871/net-misc/openssh/openssh-7.5p1.recipe#L117
https://github.com/haikuports/haikuports/blob/8e0373c81e90c2e8c1b253533fb88260975adc2f/net-misc/openssh/openssh-7.6p1.recipe#L117
https://github.com/haikuports/haikuports/blob/d5fc5c61a7525ad20c5daf0719189e65dd7f8f39/sys-libs/ncurses/ncurses-5.9.recipe#L142
https://github.com/haikuports/haikuports/blob/454940d466032a08ee262a3571d9fa1a4259bf20/sys-libs/ncurses/ncurses6-6.0.recipe#L117

</Details>